src/bin/rustfmt.rs

```diff
@@ -125,11 +125,14 @@ fn make_opts() -> Options {
         "config-help",
         "show details of rustfmt configuration options",
     );
-    opts.optopt(
+    opts.opt(
         "",
         "dump-default-config",
-        "Dumps the default configuration to a file and exits.",
+        "Dumps the default configuration to a file and exits. PATH defaults to rustfmt.toml if \
```

Comment:
this comment is incorrect?

Reply:
Added a commit to correct this in #2060.

Reply:
Whoops, thanks for noticing and fixing this.